### ISSUE TITLE
Code for untargeted attack?

### ISSUE THREAD

**Issue by joh10891** [shangtse/robust-physical-attack#1]
When I was looking through the code, I don't see a clear way to run it for an untargeted attack, though your paper and videos show you were able to do it untargeted. How can someone run it for an untargeted attack? I see TARGET_CLASS, but am uncertain how to run if I want it untargeted. Thank you, and very interesting project!

**Comment by shangtse** [comment]
In our paper, we set the target class to the true label (e.g., stop sign) and negated the cross-entropy loss. The idea is to maximize the distance (i.e., cross-entropy) between the prediction and the true class.

**Comment by joh10891** [comment]
Ah, thank you so much, and I appreciate the quick reply! So you simply changed the `nn.softmax_ctross_entropy_with_logits_v2` line to subtract the true logits from a score of 1?

**Comment by shangtse** [comment]
Correct.
